### PR TITLE
Generate report in .docx format [SCI-3537]

### PR DESCRIPTION
**Pull request by aignatov-bio**
Jira ticket: [SCI-3537](https://biosistemika.atlassian.net/browse/SCI-3537)

### What was done
Added support for reports in DOCX format

### Addons
Electronic signatures - https://gitlab.com/biosistemika/scinote/addons/electronic_signatures/merge_requests/44
EC Europa - https://gitlab.com/biosistemika/scinote/addons/ec_europa/merge_requests/34

### CODE REVIEW

File: app/services/report/docx_action/experiment.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# rubocop:disable  Style/ClassAndModuleChildren
+module Report::DocxAction::Experiment
```
Comment (Ducz0r) [inline app/services/report/docx_action/experiment.rb]: Do we have any good practice about how to name services? I'd say it's probably better to use e.g. `app/services/reports/generate_docx/...` or something like that.

Comment (Ducz0r) [inline app/services/report/docx_action/experiment.rb]: Do we have any good practice about how to name services? I'd say it's probably better to use e.g. `app/services/reports/generate_docx/...` or something like that.

Comment (aignatov-bio) [reply app/services/report/docx_action/experiment.rb]: In this approach we receive very complex names with duplicates naming. 
My structure expect, that we have main folder for reports. One main class for each type of report and folder with additional modules and classes. I will optimize it little bit.

Comment (Ducz0r) [reply app/services/report/docx_action/experiment.rb]: :+1: In any case, other folders use plural, so at least it should be `services/reports/...`

**Comment by Ducz0r** [comment]
The TravisCI build is failing because Travis didn't update its branches to `master` after this PR has been re-targeted to `master` from `rc/16.5`.

**Pull request merged**